package.json

```diff
@@ -13,29 +13,29 @@
     "@tinymce/tinymce-vue": "^1.1.0",
     "chart.js": "^2.8.0",
     "debounce": "^1.2.0",
-    "tinymce": "^4.9.4",
+    "tinymce": "^4.9.6",
     "v-tooltip": "^2.0.2",
     "vue": "^2.6.10",
-    "vue-autosuggest": "^2.0.1",
+    "vue-autosuggest": "^2.0.4",
     "vue-chartjs": "^3.4.2",
     "vue-color": "^2.7.0",
-    "vue-scrollto": "^2.15.0",
+    "vue-scrollto": "^2.17.1",
     "vue-slider-component": "^2.8.16",
-    "vue2-dropzone": "^3.5.9",
-    "vuedraggable": "^2.20.0"
+    "vue2-dropzone": "^3.6.0",
+    "vuedraggable": "^2.23.2"
   },
   "devDependencies": {
-    "@vue/cli-plugin-babel": "^3.7.0",
-    "@vue/cli-plugin-eslint": "^3.7.0",
-    "@vue/cli-service": "^3.7.0",
+    "@vue/cli-plugin-babel": "^3.11.0",
+    "@vue/cli-plugin-eslint": "^3.11.0",
+    "@vue/cli-service": "^3.11.0",
     "@vue/eslint-config-prettier": "^4.0.1",
-    "babel-eslint": "^10.0.1",
+    "babel-eslint": "^10.0.3",
     "element-resize-event": "^3.0.3",
     "eslint": "^5.16.0",
-    "eslint-plugin-vue": "^5.2.2",
-    "less": "^3.0.4",
+    "eslint-plugin-vue": "^5.2.3",
+    "less": "^3.10.3",
     "less-loader": "^4.1.0",
-    "lint-staged": "^8.1.6",
+    "lint-staged": "^8.2.1",
```

Comment:
There are a lot of version bumps here. Usually when I run an update like this, I do some manual checking to try to identify any regressions... We also need to keep the versions here in sync with the version in the UI Library.

So I'd suggest either not including these changes in a commit (I'm happy to run an update whenever if you'd like) or making sure that the same changes happen in UI Library and doing some testing on it...

Reply:
OK, happy to remove these!